lib/job.js

```diff
@@ -193,13 +193,25 @@ class Job extends BaseModel {
 
     /**
      * Return latest build that belong to this job
-     * @param  {Object}   [config]                  Configuration object
-     * @param  {String}   [config.status]           Return latest build with this status
-     * @return {Promise}                            Lastest build
+     * @param  {Object}   [config]              Configuration object
+     * @param  {String}   [config.status]       Return build with this status
+     * @param  {String}   [config.position]     Return build with this position relative to latest build
+     * @return {Promise}                        Lastest build
      */
     getLatestBuild(config = {}) {
-        return this.getBuilds({ status: config.status })
-            .then(latestBuilds => latestBuilds[0] || {});
+        const { position, status } = config;
+        // passing string instead of number to retain old behavior
+        const parsedPoistion = Number(position);
+        let relativePosition = -1;
```

Comment:
Not sure if this helps with the developer experience though. 🤷‍♂ 

```
        // default revision in asscending order
        // i.e. [0, 1, 2, 3, 4, 5]
        // 0 is the latest, which is 0
        // 1 is second-to-last, which is 1
        // -1 is the first, which is 5

        let absolutePosition = 0;

        if (relativePosition === 'latest') {
          absolutePosition = 0;
        } else {
          const parsedPoistion = Number(relativePosition);

          if (!isNaN(parsedPoistion) {
            if (parsedPoistion < 0) {
              // totalNumberOfRevision, not sure if we have it or not.
              absolutePosition = totalNumberOfRevision + parsedPoistion;
            } else {
              absolutePosition = parsedPoistion;
            }
          }
        }
```

Reply:
Looks like `totalNumberOfRevision` is not gonna be available, negative query is not gonna be implemented for the time being.